### ISSUE TITLE
WebStorm 2020: wrong result

### ISSUE THREAD

**Issue by Snaptags** [oliversalzburg/which-webstorm#1]
Is this project still active? I just noticed that it does not work as expected using various WebStorm 2020 versions. On my Windows machine I have
```
d----        24.07.2020     19:02        1   WebStorm 2020.1
d----        24.07.2020     19:00        1   WebStorm 2020.1.4
d----        04.08.2020     10:00        1   WebStorm 2020.2
```
and would expect to get the 2020.2 path. But `which-webstorm` returns 2020.1.4.

**Comment by oliversalzburg** [comment]
It's not really alive, because I no longer use WebStorm, but we can make it work regardless 😊

Currently `which-webstorm` will return just any folder that contains a valid WebStorm executable. Maybe all valid candidates should be sorted and it should pick the "last" one?

**Comment by Snaptags** [comment]
Thanks, I'll add a PR then.

**Comment by oliversalzburg** [comment]
Fixed by #2
